code/modules/client/client_colour.dm

```diff
@@ -108,6 +108,10 @@
 	colour = "#cccccc"
 
 
+/datum/client_colour/eye_colour/monochrome
```

Comment:
There's already a monochrome type just below this and there doesnt look like there's any other `/datum/client_colour/eye_colour` types

Reply:
Client colour doesn't use sources, so if I just reuse the other one it will delete the monochrome client color for people with the quirk.